### PR TITLE
V0.0.2 get unspent rpc

### PR DESCRIPTION
**Pull request by matt-auckland**

### CODE REVIEW

File: server/src/lib/rpc/client.js [hunk 1]
```diff
@@ -40,4 +41,14 @@ rpc.nav.getNewAddress = async () => {
   }
 }
 
+rpc.nav.listUnspentTx = async () => {
```
Comment (ro-savage) [inline server/src/lib/rpc/client.js:44]: Why do we need this?
Can't we just use `rpc.listUnspent()` directly? Why does it need a wrapper?

File: server/test/lib.rpc.client.spec.js [hunk 2]
```diff
@@ -147,4 +148,33 @@ describe('[Rpc]', () => {
       })
     })
   })
+
+  describe('(nav.listUnspentTx)', () => {
```
Comment (ro-savage) [inline server/test/lib.rpc.client.spec.js:152]: If we don't have a wrapper, we don't need this test.

**Review by ro-savage — CHANGES_REQUESTED** [review]
I am not sure the purpose of this PR?
Why do we need to write a wrapper around `rpc.listUnspent()` ?

**Comment by matt-auckland** [comment]
actually you're right 🤦‍♂️ 
I'm closing this.

**Pull request closed without merging**